src/recon_buildblock/Parallelproj_projector/ParallelprojHelper.cxx

```diff
@@ -73,7 +73,7 @@ detail::ParallelprojHelper::ParallelprojHelper(const ProjDataInfo& p_info, const
   Bin bin;
   LORInAxialAndNoArcCorrSinogramCoordinates<float> lor;
   LORAs2Points<float> lor_points;
-  const float radius = p_info.get_scanner_sptr()->get_inner_ring_radius();
+  const float radius = p_info.get_scanner_sptr()->get_max_FOV_radius() + p_info.get_scanner_sptr()->get_average_depth_of_interaction();
```

Comment:
I think still to remove the DOI

Reply:
If I remove the DOI, it will crop parts of the sinogram though - is that what we want?

Reply:
well, it will crop the part where they will never be any activity. no? (no activity inside the detector)

Reply:
![image](https://user-images.githubusercontent.com/87262701/194569978-fe3a59de-8d74-44df-a6c3-4e27741bc735.png)
![image](https://user-images.githubusercontent.com/87262701/194570041-ea595e62-659d-4be9-88ee-7b1914745523.png)
Yes, I agree. But strangely enough it does impact the way the resulting sinograms look (first: with DOI; second: without DOI). I now removed the "+ DOI" in the code, but would still be good to understand why the sinograms differ.

Reply:
This has been discussed, and it is actually caused by LOR's along one detector module that remain behind the front edge of the crystal. Without modelling DOI they are obviously cropped, whereas they are included in the outermost parts of the sinogram when including the DOI for the definition of the LOR's. Since these LOR's are not relevant for the image reconstruction, they can be set to zero as in the second plot.